toolkit/tools/graphPreprocessor/graphPreprocessor.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"os"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+	"microsoft.com/pkggen/internal/exe"
+	"microsoft.com/pkggen/internal/logger"
+	"microsoft.com/pkggen/internal/pkggraph"
+)
+
+const (
+	allGoalNodeName = "ALL"
+)
+
+var (
```

Comment:
Nit: not used anymore.

```suggestion
var (
```